lib/core/src/display/scene/htmlobject.rs

```diff
@@ -0,0 +1,49 @@
+use super::Object;
+use crate::system::web::{create_element_as, Result};
+use nalgebra::Vector3;
+use web_sys::HtmlElement;
+
+#[derive(Shrinkwrap)]
+#[shrinkwrap(mutable)]
+pub struct HTMLObject {
+    #[shrinkwrap(main_field)]
+    pub object: Object,
+    pub element: HtmlElement,
+    pub dimension: Vector3<f32>,
```

Comment:
I believe these are `dimensions`, not `dimension`, right?

Reply:
Yes, fixing it.